Formula/r/regclient.rb

```diff
@@ -0,0 +1,26 @@
+class Regclient < Formula
+  desc "Docker and OCI registry tools"
+  homepage "https://github.com/regclient/regclient"
+  url "https://github.com/regclient/regclient/archive/refs/tags/v0.5.3.tar.gz"
+  sha256 "2ea493c3ed6fe24416e8174e0298886cc0c05155e7258365848b7566e807bc19"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # "$(BINARIES)" targets depends on "vendor", but it's not marked as such in the Makefile.
+    # without this, the build fails with:
+    # ... is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
```

Comment:
please file an upstream issue to fix vendor issue.

in fact, with go modules build, there is no need to have vendor folders at all

Reply:
Upstream have just merged the the [parallel make fix](https://github.com/regclient/regclient/pull/588).

I've no idea about go modules build and vendor folders since I'm not a go user. I'm not sure whether it should/how to report to upstream.

Reply:
let me check a bit.

Reply:
that is because they have this in makefile

```
.PHONY: vendor
vendor: ## Vendor Go modules
	go mod vendor

.PHONY: binaries
binaries: vendor $(BINARIES) ## Build Go binaries
```